content/python/concepts/tuples/terms/count/count.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.count()'
+Description: 'Returns the number of occurrences of a specific value in a tuple.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Tuples'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.count()`** method returns the number of occurrences of a specific value in a tuple.
+
+## Syntax
+
+```pseudo
+tuple.count(value)
+```
+
+The `value` parameter is required. If the `value` does not exist in the `tuple`, then `0` is returned.
+
+## Example
+
+The following example showcases the `.count()` method:
+
+```py
+my_tuple = (2, 4, "6", "2", 8, "8", "2")
+
+print(my_tuple.count(2))
+# Output: 1
+```
+
+The `.count()` method is type-sensitive. This is why the snippet above returned `1` (for one occurrence of the number `2`, excluding the string representation of `"2"`).
```

Comment:
I'm wondering if we can include that this method is type-sensitive in the intro, then just reinforce that idea here by starting out the paragraph here like:

Since the method is type-sensitive...